hybridse/src/codegen/udf_ir_builder_test.cc

```diff
@@ -1007,6 +1007,14 @@ TEST_F(UdfIRBuilderTest, degrees) {
     CheckUdf<double, double>(udf_name, -90.0, -pi/2);
     CheckUdf<Nullable<double>, Nullable<double>>(udf_name, nullptr, nullptr);
 }
+
+TEST_F(UdfIRBuilderTest, char_length_udf_test) {
+    auto udf_name = "char_length";
+    CheckUdf<int32_t, StringRef>(udf_name, 10, StringRef("Spark SQL "));
```

Comment:
ensure special char like `\n` is correctly counted

Reply:
ok